helm/operator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A Helm chart for MinIO Operator
 name: operator
-version: 4.4.28
+version: 4.5.0
```

Comment:
do not make these changes @funkypenguin changing versions please leave this to maintainers..

Reply:
Got it. I'll revert that.